code/game/objects/items/devices/laserpointer.dm

```diff
@@ -275,14 +275,12 @@
 			continue
 		if(target_felinid.body_position == STANDING_UP)
 			target_felinid.setDir(get_dir(target_felinid, targloc)) // kitty always looks at the light
-			//SKYRAT EDIT REMOVAL BEGIN (removes forced felinid movement from laserpointers, also fixes the longstanding windoor negation glitch)
-			/* if(prob(effectchance * diode.rating))
+			if(prob(effectchance * diode.rating))
```

Comment:
dont you hvae to do a bubber edit instead of just reverting it

Reply:
correct